modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -12,15 +12,24 @@ The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage
 The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the service account and Workload Identity Federation authentication type. For additional requirements that apply to the service account authentication type only, see _Service account authentication type procedure_. For additional requirements that apply to the Workload Identity Federation authentication type only, see _Workload Identity Federation authentication type procedure_.
 ====
 
-[WARNING]
-====
-To use {product-title} in your GCP project, the following GCP organizational policy constraints cannot be in place:
+.Prerequisites
+
+Before using {product-title} in your GCP project, confirm that the following organizational policy constraints are configured correctly where applicable:
 
-* `constraints/iam.allowedPolicyMemberDomains` (This policy constraint is supported only if Red Hat's `DIRECTORY_CUSTOMER_IDs C02k0l5e8` and `C04j7mbwl` are included in the allow list. Use this policy constraint with caution).
+* `constraints/iam.allowedPolicyMemberDomains`
+** This policy constraint is supported only if Red Hat's DIRECTORY CUSTOMER ID's `C02k0l5e8` and `C04j7mbwl` are included in the allow list.
```

Comment:
```suggestion
** This policy constraint is supported only if Red {nbsp} Hat's DIRECTORY CUSTOMER ID's `C02k0l5e8` and `C04j7mbwl` are included in the allowlist.
```

Reply:
Should be a non breaking space between 'Red' and 'Hat'. I can't seem to add a suggestion for lines 9 to include this also.

Reply:
The "DIRECTORY CUSTOMER ID’s" does not need to be capitalized.
> Red Hat's Directory Customer ID's